acceptance/tests/storeconfigs/basic_collection.rb

```diff
@@ -16,20 +16,17 @@
 
   manifest << "\nNotify <<| |>>"
 
-  tmpdir = master.tmpdir('storeconfigs')
-
-  manifest_file = File.join(tmpdir, 'site.pp')
-
-  create_remote_file(master, manifest_file, manifest)
-
-  on master, "chmod -R +rX #{tmpdir}"
-
+  manifest_path = create_remote_site_pp(master, manifest)
   with_puppet_running_on master, {
     'master' => {
       'autosign' => 'true',
-      'manifest' => manifest_file
-    }} do
+    },
+    'main' => {
+      'environmentpath' => manifest_path
+    }
+  } do
 
+    bounce_service( master, master['puppetservice'], 10 )
```

Comment:
I see a proliferation of the bounce_service after with_puppet_running_on, this is a little strange any specific reason for this?